ocaml/typing/layouts.ml

```diff
@@ -993,16 +996,21 @@ module Layout = struct
 
   let equate = equate_or_equal ~allow_mutation:true
 
+  let score_reason reason =
```

Comment:
Good function. Please add a comment describing what it does and why it's a good idea.